packages/web-components/package.json

```diff
@@ -58,9 +57,13 @@
     "@types/chai-spies": "^1.0.1",
     "@types/karma": "^5.0.0",
     "@types/mocha": "^7.0.2",
+    "@typescript-eslint/eslint-plugin": "4.22.0",
```

Comment:
I'll move those to single version policy as well in separate PR, once thats merged this can be removed as well